bin/cli.js

```diff
@@ -288,18 +288,24 @@ For more information, see https://webpack.js.org/api/cli/.`);
 				}).apply(compiler);
 			}
 			if (outputOptions.infoVerbosity === "verbose") {
-				if (argv.w) {
-					compiler.hooks.watchRun.tap("WebpackInfo", compilation => {
+				const compilers = compiler.compilers ? compiler.compilers : [compiler];
+
+				compilers.map(compiler => setUpHookForCompilation(compiler, argv));
+			}
+			function setUpHookForCompilation(compiler, args) {
+				if (args.w) {
+					compiler.hooks.watchRun.tap("webpackInfo", compilation => {
 						const compilationName = compilation.name ? compilation.name : "";
 						console.error("\nCompilation " + compilationName + " starting…\n");
 					});
 				} else {
-					compiler.hooks.beforeRun.tap("WebpackInfo", compilation => {
+					compiler.hooks.beforeRun.tap("webpackInfo", compilation => {
 						const compilationName = compilation.name ? compilation.name : "";
 						console.error("\nCompilation " + compilationName + " starting…\n");
 					});
 				}
-				compiler.hooks.done.tap("WebpackInfo", compilation => {
+				compiler.hooks.done.tap("webpackInfo", stats => {
```

Comment:
All hooks should be used package name, because it can potential break other plugins, use `WebpackCLI` (for all hooks), if you found other hooks with invalid name in our code based change it too

Reply:
Okay.